iianmelo/citigames-squadian
# Feat/component/button

What I did
- created the Button folder in components
- added the button ('entrar'/'sair')
- created the variable 'greenHover', 'grayButton',  'grayButtonHover', 'redButton' and 'redButtonHover' for the button color (style/global and tailwind.config)
- Implemented the font-family 'Barlow' in style/global and tailwind.config
- On App/page there are roomStatus tests for the button cases

How to test
	1	Open the terminal and go to the folder of this project
	2	In terminal:
	i	git checkout main
	ii	git pull
	iii	git checkout <feat/component/button>
	iv	git pull origin <feat/component/button>
	v	yarn install
	vii	yarn dev
	3	In your browser go to http://localhost:3000
	4	Check if the component looks just like the mockup in different screen sizes
	5	In VScode open the relative path of the test page
	6	Test the components/pages that were made
	7	Read the changed files section on GitHub
	8	Test the things that you think are worth testing, even the ones that are not in this description
